src/components/SideMenu/SideMenu.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { ReactNode } from 'react';
+
+import classNames from 'classnames/bind';
+
+import { ReactComponent as logo } from '@assets/main_logo.svg';
+import { Drawer } from '@components/Drawer';
+import { SvgIcon } from '@components/SvgIcon';
+
+import styles from './SideMenu.module.css';
+
+const cn = classNames.bind(styles);
+
+export enum AnchorEnum {
+ left = 'left',
+ right = 'right',
+}
+
+export interface SideMenuProps {
+  anchor?: AnchorEnum;
+  children: ReactNode;
+  onClose: VoidFunction;
+  open: boolean;
+}
+
+export function SideMenu({
+  anchor = AnchorEnum.left,
+  children,
+  onClose,
+  open,
+}: SideMenuProps) {
+  return (
+    <Drawer
+      className={cn('side-menu')}
+      paperClassName={cn('side-menu__paper')}
+      anchor={AnchorEnum.left}
+      open={open}
+      onClose={onClose}
+    >
+      <div className={cn('side-menu__header')}>
+        <SvgIcon
+          icon="close"
+          className={cn('side-menu__close-icon', {
+            'side-menu__close-icon--visible': anchor === AnchorEnum.left,
+          })}
+          onClick={onClose}
+        />
+
+        <SvgIcon icon={logo} className={cn('side-menu__logo')} viewBox="" />
```

Comment:
Can we maybe add viewBox or remove the viewBox if it's gonna be an empty string. Please do let me know here